### PR TITLE
29 Optimise and Improve Code Base

### PR DESCRIPTION
**Pull request by nonsobiose**
- Fixed issue with blank IOS map
- Removed coords.js (No need to call home)
- Added logs.js to log events
- Added utils.js for log implementation
- Removed irrelevant code

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -25,183 +22,52 @@ export default class App extends Component {
...
...N":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":170000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224702":{"PAON":"4","SAON":"BASEMENT FLAT","Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":147000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224703":{"PAON":"4","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":160000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224704":{"PAON":"5B","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":280000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224705":{"PAON":"6","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":500000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224706":{"PAON":"7","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":502000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224707":{"PAON":"8","SAON":"FLAT 1","Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":180000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224708":{"PAON":"8","SAON":"FLAT 3","Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":245000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224709":{"PAON":"8","SAON":"FLAT 2","Street"
```
Comment (bcarter97) [inline App.js]: Do we need a console write? Perhaps something to notify the user, as the app requires it for core functionality

Comment (bcarter97) [inline App.js]: Failed to retrieve geolocation, not retrieved in error

Comment (nonsobiose) [reply App.js]: These lines of code have been removed. That's why they have been highlighted in red.

Comment (nonsobiose) [reply App.js]: These lines of code have been removed. That's why they have been highlighted in red.

Comment (bcarter97) [inline App.js]: These lines

File: ios/MyAwesomeApp/AppDelegate.m [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 #import <React/RCTBridge.h>
 #import <React/RCTBundleURLProvider.h>
-#import <React/RCTRootView.h>
+#import <React/RCTRootVizew.h>
```
Comment (georgeracu) [inline ios/MyAwesomeApp/AppDelegate.m]: Is this a typo? Vizew vs View

Comment (nonsobiose) [reply ios/MyAwesomeApp/AppDelegate.m]: That was a typo. Just fixed it.

**Pull request merged**